src/components/Popup/MainBody.vue

```diff
@@ -90,6 +90,87 @@ export default Vue.extend({
     clearFilter() {
       this.$store.dispatch("accounts/clearFilter");
     },
+    isEntryVisible(entry: OTPEntry, shouldFilter: boolean, filter: boolean) {
+      return (
+        this.isSearchedEntry(entry) &&
+        (entry.pinned || !shouldFilter || !filter || this.isMatchedEntry(entry))
+      );
+    },
+    getTabindex(
+      entry: OTPEntry,
+      entries: OTPEntry[],
```

Comment:
entries, shouldFilter, and filter are all computed properties and don't need to be passed as arguments.